web/templates/default_templates/slides/manual-calendar/manual-calendar.scss

```diff
@@ -16,7 +16,7 @@
       @extend %show-placeholder-text;
     }
 
-    font-size: 100%;
+    font-size: 110%;
```

Comment:
Kommentar - især fordi 100% kan man umiddelbart godt acceptere, men 110% lyder som om at man var nødsaget til at håndtere et eller andet issue - og det ville være godt at vide hvad var for et.